final/src/main/java/ua/epam/spring/hometask/service/impl/AuditoriumServiceImpl.java

```diff
@@ -50,9 +50,8 @@
      */
     @Nonnull
     @Override
-    public Set<Long> getAllSeats(long numberOfSeats) {
-        return LongStream.range(1, numberOfSeats + 1)
-                .boxed()
-                .collect(Collectors.toSet());
+    public Page<Auditorium> getAllSeats(int page, int numberOfSeats) {
+        PageRequest request = PageRequest.of(page, numberOfSeats + 1);
+        return auditoriumRepository.findAll(request);
```

Comment:
Think about performance. Reduce calls to database when unnecessary. I'd recommend to retrieve all data in one call to db.